lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -752,14 +753,15 @@ export abstract class CacheManager implements ICacheManager {
      * @param account
      * @param familyRT
      */
-    readRefreshTokenFromCache(clientId: string, account: AccountInfo, familyRT: boolean): RefreshTokenEntity | null {
+    readRefreshTokenFromCache(clientId: string, account: AccountInfo, familyRT: boolean, hash?: string): RefreshTokenEntity | null {
```

Comment:
Same as above re: multiple hashes

Reply:
This will be needed when we start supporting RT flow. Removing as it is not needed now.